frontend/src/lib/api/access-list.ts

```diff
@@ -91,14 +92,14 @@ class AccessListService {
   /**
    * Refresh the cached configuration
    */
-  async refreshConfig(): Promise<any> {
+  async refreshConfig(): Promise<VyOSResponse> {
     return apiClient.post("/vyos/config/refresh");
   }
 
   /**
    * Execute batch operations
    */
-  async batchConfigure(request: AccessListBatchRequest): Promise<any> {
+  async batchConfigure(request: AccessListBatchRequest): Promise<VyOSResponse> {
     const result = await apiClient.post("/vyos/access-list/batch", request);
```

Comment:
If you promise <VyOSResponse> you also have to update the `apiClient.post

```suggestion
    const result = await apiClient.post<VyOSResponse>("/vyos/access-list/batch", request);
```

Reply:
Sorry, my mistake, I missed that.